src/Workspaces/Core/Portable/NamingStyles/Serialization/AccessibilityExtensions.cs

```diff
@@ -21,5 +21,30 @@ internal static Accessibility FromXElement(XElement accessibilityElement)
         {
             return (Accessibility)Enum.Parse(typeof(Accessibility), accessibilityElement.Value);
         }
+
+        private static Accessibility GetAccessibility(ISymbol symbol)
+        {
+            for (var currentSymbol = symbol; currentSymbol != null; currentSymbol = currentSymbol.ContainingSymbol)
+            {
+                switch (currentSymbol.Kind)
+                {
+                case SymbolKind.Namespace:
+                    return Accessibility.Public;
+
+                case SymbolKind.Parameter:
+                case SymbolKind.TypeParameter:
+                    continue;
+
+                case SymbolKind.Method when ((IMethodSymbol)currentSymbol).MethodKind == MethodKind.LocalFunction:
```

Comment:
What about parameters of lambdas & anonymous methods?
They should be local, so this is likely missing another case for `MethodKind`.

Reply:
I added the following tests to `TestPascalCaseSymbol_ExpectedSymbolAndAccessibility` and fixed the condition here.

```
[InlineData("void Outer() { void Inner(int [|m|]) {} }", "void Outer() { void Inner(int M) {} }", SymbolKind.Parameter, Accessibility.NotApplicable)]
[InlineData("void Outer() { System.Action<int> action = [|m|) => {} }", "void Outer() { System.Action<int> action = M => {} }", SymbolKind.Parameter, Accessibility.NotApplicable)]
[InlineData("void Outer() { System.Action<int> action = ([|m|]) => {} }", "void Outer() { System.Action<int> action = (M) => {} }", SymbolKind.Parameter, Accessibility.NotApplicable)]
[InlineData("void Outer() { System.Action<int> action = (int [|m|]) => {} }", "void Outer() { System.Action<int> action = (int M) => {} }", SymbolKind.Parameter, Accessibility.NotApplicable)]
[InlineData("void Outer() { System.Action<int> action = delegate (int [|m|]) {} }", "void Outer() { System.Action<int> action = delegate (int M) {} }", SymbolKind.Parameter, Accessibility.NotApplicable)]
```

I'll wait to see if there are other tests I need to add before pushing this update.

Reply:
➡️ Fixed in 40ded3c5678be13246ac7a017864a24cf0a890f5